src/icalendar/__init__.py

```diff
@@ -44,3 +44,9 @@
 
 # Switching the timezone provider
 from icalendar.timezone import use_pytz, use_zoneinfo
+
+
+__all__ = ['Calendar', 'Event', 'Todo', 'Journal', 'Timezone', 'TimezoneStandard', 'TimezoneDaylight', 'FreeBusy', 'Alarm',
+          'ComponentFactory', 'vBinary', 'vBoolean', 'vCalAddress', 'vDatetime', 'vDate', 'vDDDTypes', 'vDuration',
+          'vFloat, 'vInt', 'vPeriod','vWeekday', 'vFrequency', 'vRecur', 'vText', 'vTime', 'vUri', 'vGeo', 'vUTCOffset',
```

Comment:
Typo fix.

This also needs a change log entry.

```suggestion
          'vFloat', 'vInt', 'vPeriod','vWeekday', 'vFrequency', 'vRecur', 'vText', 'vTime', 'vUri', 'vGeo', 'vUTCOffset',
```

Reply:
I used github web so I couldn't spot the typo before pr. I've resolved it
sorry about that